### PR TITLE
feat: migrate the home page

### PR DESCRIPTION
**Pull request by gregatgoogle**
image

### CODE REVIEW

File: lib/menu.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import type {
+  MenuOverlayItem,
+  MenuOverlayStrings,
+} from '@ircsignpost/signpost-base/dist/src/menu-overlay';
+
+export function getMenuItems(strings: MenuOverlayStrings): MenuOverlayItem[] {
```
Comment (ishparii) [inline lib/menu.ts:6]: Do we need a TODO here to implement this for every site or is it just going to be migrated separately?

I think we should be able to just copy the function from Beporsed as is, wdyt?

File: pages/index.tsx [hunk 2]
```diff
@@ -1,72 +1,118 @@
+import HomePage, {
+  HomePageStrings,
+} from '@ircsignpost/signpost-base/dist/src/home-page';
+import { MenuOverlayItem } from '@ircsignpost/signpost-base/dist/src/menu-overlay';
+import { ServiceMapProps } from '@ircsignpost/signpost-base/dist/src/service-map';
+import { ZendeskCategory } from '@ircsignpost/signpost-base/dist/src/zendesk';
+import {
```
Comment (ishparii) [inline pages/index.tsx:7]: Add a TODO to swap this dependency

**Review by ishparii — APPROVED** [review]

File: pages/index.tsx (hunk 2 above)
Comment (gregatgoogle) [reply pages/index.tsx:8]: Done

File: lib/menu.ts (hunk 1 above)
Comment (gregatgoogle) [reply lib/menu.ts:7]: I added a TODO, and I propose @EOkoyomon that you use lib/menu for this. I leave any special modification of the menu for later commits. This one is only for an MVP of the home page.

**Pull request merged**